src/js/moderator/productModerator.js

```diff
@@ -0,0 +1,53 @@
+import { CONFIRM_MESSAGE, EVENT_TYPE } from "../constant";
+import ProductProcessMachine from "../domain/productProcessMachine";
+import ProductPageView from "../ui/productPageView";
+import { on } from "../util/event";
+
+class ProductModerator {
+  constructor() {
+    this.productProcessMachine = new ProductProcessMachine();
+    this.productPageView = new ProductPageView();
+    on(window, EVENT_TYPE.ADD, (e) => this.addProduct(e.detail));
+    on(window, EVENT_TYPE.DELETE, (e) => this.deleteProduct(e.detail));
+    on(window, EVENT_TYPE.EDIT, (e) => this.updateProduct(e.detail));
+  }
+
+  init = () => {
+    this.productPageView.renderInputForm();
+    this.productPageView.initDOMS();
```

Comment:
코드 순서상 Init 이 먼저 된 다음에 render 를 해야겠죠?

```suggestion
    this.productPageView.initDOMS();
    this.productPageView.renderInputForm();
```

Reply:
지금보니 정말...순서가 말도 안되네요 😭 
이 부분은 코드에서 Dom Element를 다루는 방법을 바꾸면서 수정되었습니다 :)
감사합니다!